backend/src/routes/Authentifications.ts

```diff
@@ -0,0 +1,22 @@
+import { Router } from "express";
+import AuthentificationsController from '../controllers/AuthentificationsController';
```

Comment:
typo should be AuthenticationController
without the `f` and trailing `s`

Reply:
fixed